math/libcerf/files/patch-powerpc.diff

```diff
@@ -0,0 +1,15 @@
+--- CMakeLists.txt.orig	2023-01-10 16:10:08.000000000 +0800
++++ CMakeLists.txt	2023-06-26 02:28:37.000000000 +0800
+@@ -76,7 +76,11 @@
+     if (CMAKE_CXX_COMPILER_ID STREQUAL "GNU" OR CMAKE_C_COMPILER_ID STREQUAL "GNU" )
+         option(PORTABLE "Build a portable binary without host-specific optimization" OFF)
+         if(NOT PORTABLE)
```

Comment:
@barracuda156 I suggest to switch on `PORTABLE` here, otherwice it an artifact from build bot might not work on user's machine.

Reply:
UPD. In fact no need, since that option is only used for GCC, and Macports has no buildbots using GCC:
https://jugit.fz-juelich.de/mlz/libcerf/-/blob/main/CMakeLists.txt#L76